client/css/style.css

```diff
@@ -958,7 +958,7 @@ kbd {
 }
 
 #chat .time {
-	color: #ddd;
+	color: #777;
```

Comment:
Before | After
--- | ---
<img width="1011" alt="screen shot 2017-08-12 at 21 17 34" src="https://user-images.githubusercontent.com/113730/29245296-e377735c-7fa3-11e7-95ee-c065ccbdcc59.png"> | <img width="1010" alt="screen shot 2017-08-12 at 21 18 43" src="https://user-images.githubusercontent.com/113730/29245299-e577f938-7fa3-11e7-9216-2cf60420a325.png">

I totally understand that this is not contrasted enough, but with `#777` there is pretty much no difference with normal text. Since time appears with every. single. message, that *After* view is very crowded and harsh...

I would leave them as is, until someday maybe if we add a "high contrast mode" option.